dev/com.ibm.ws.jaxrs.2.0_fat/publish/servers/com.ibm.ws.jaxrs.fat.options/server.xml

```diff
@@ -0,0 +1,13 @@
+<server>
+    <featureManager>
+        <feature>jaxrs-2.0</feature>        
+        <feature>componenttest-1.0</feature>
+    </featureManager>
+
+  	<include location="../fatTestPorts.xml"/>
+  	
+	<javaPermission className="java.net.SocketPermission" name="*" actions="connect,resolve"/>
+    <javaPermission className="java.util.PropertyPermission" name="org.apache.commons.logging.*" actions="read"/>
+    <javaPermission className="java.lang.RuntimePermission" name="getClassLoader"/> 
```

Comment:
These don't look right.  Are you seeing Java 2 security failures without these? If so, I think we might need to add doPriv blocks in the runtime code to address - I don't see anywhere in the test case (application) code where it is getting a classloader or looking up an org.apache.* system property.